Lombiq.Tests.UI/MonkeyTesting/MonkeyTestingOptions.cs

```diff
@@ -110,25 +110,26 @@ public sealed class MonkeyTestingOptions
         };
 
         /// <summary>
-        /// Adds the login page (/Login) to the URL filters.
+        /// Adds the login page (/Login) to the URL filters (i.e. monkey testing will not happen on that page).
```

Comment:
A comment mellett még jobb lenne, ha a metódus nevek is ezt tükröznék, mondjuk inkább "ExcludeLoginPageFromMonkeyTesting".